ext/ManifoldsOrdinaryDiffEqExt.jl

```diff
@@ -45,13 +44,25 @@ function solve_exp_ode(
     q = sol.u[end][(n + 1):(2 * n)]
     return q
 end
-# also define exp! for metric manifold anew in this case
-function exp_fused!(
-        ::TraitList{IsMetricManifold}, M::AbstractDecoratorManifold, q, p, X, t::Number;
-        kwargs...
+
+function ManifoldsBase.retract_exp_ode!(
```

Comment:
Since we're making a breaking release here, I think we could finally remove this solver? `solve_chart_exp_ode` should be the replacement.